proto/storage/deployment.proto

```diff
@@ -49,6 +49,8 @@ message Deployment {
   float risk_score = 29; // @gotags: search:"Deployment Risk Score,hidden" policy:",ignore"
 
   reserved 30; // was tags
+
+  bool platform_component = 35; // @gotags: search:"Platform Component"
```

Comment:
Update the next tag comment at the top.  Part of me wants to suggest putting these in numerical order as that would be far more maintainable.  but probably safer to leave it be.